spec/containers/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3'
+services:
+  xenial:
+    build:
+      context: .
+      dockerfile: Dockerfile.xenial
+    ports:
+      - "1122:22"
+    # --security-opt seccomp=unconfined --tmpfs /run --tmpfs /run/lock -v /sys/fs/cgroup:/sys/fs/cgroup:ro
```

Comment:
Do you still need this comment?

Reply:
It's just a handy hint if you want to run the container from the command line

Reply:
Got it 👍